docs/en/reference/working-with-associations.rst

```diff
@@ -32,62 +32,66 @@ information about its type and if it's the owning or inverse side.
 .. code-block:: php
 
     <?php
-    /** @Entity */
+    #[Entity]
     class User
     {
-        /** @Id @GeneratedValue @Column(type="string") */
-        private $id;
+        #[Id]
+        #[GeneratedValue]
+        #[Column(type: 'string')]
+        private string $id;
 
         /**
          * Bidirectional - Many users have Many favorite comments (OWNING SIDE)
          *
-         * @ManyToMany(targetEntity="Comment", inversedBy="userFavorites")
-         * @JoinTable(name="user_favorite_comments")
+         * @var Collection<int, Comment>
```

Comment:
The key doesn't really matter here, and I'm not sure it should be `int` either. I don't know if we should recommend omitting it, or using `array-key`, etc.

Reply:
The key does matter if you use the `indexedBy` option. If you don't, `int` is correct.